### DISCUSSION TITLE
fullyQualifiedName not calculated right when using wildcard imports

### DISCUSSION THREAD

**Discussion by caskolkm** [LemonAppDev/konsist#1629] [Report a Problem]
### Tell us what version of Konsist you use.

0.17.3

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Describe the problem

Given the below Spring RestController

```
package test.api

import org.springframework.web.bind.annotation.* 

@RestController
@RequestMapping("/api/v1")
class MyController {

    @GetMapping("string")
    fun string(): String = ""
}
```

The fullyQualifiedName(s) of the annotations of the MyController are not fullyQualifiedNames. They are just "RestController" and "RequestMapping". Therefor hasAnnotationOf doesn't work.

If i get rid of the wildcard import, it is working.

### Actual behaviour

This test fails
```
    @Test
    fun `all classes in the package should be RestControllers`() {
        Konsist
            .scopeFromProduction()
            .classes()
            .withPackage("..api")
            .assertTrue(strict = true) {
                it.hasAnnotationOf(RestController::class)
            }
    }
```

### Expected behaviour

This test should succeed
```
    @Test
    fun `all classes in the package should be RestControllers`() {
        Konsist
            .scopeFromProduction()
            .classes()
            .withPackage("..api")
            .assertTrue(strict = true) {
                it.hasAnnotationOf(RestController::class)
            }
    }
```

**Comment by igorwojda** [comment]
Will be fixed in `18.0.0`
https://github.com/LemonAppDev/konsist/pull/1813
